services/src/main/java/org/keycloak/broker/oidc/mappers/AbstractJsonUserAttributeMapper.java

```diff
@@ -125,9 +125,9 @@ public void preprocessFederatedIdentity(KeycloakSession session, RealmModel real
 		}
 		attribute = attribute.trim();
 
-		String value = getJsonValue(mapperModel, context);
+		Object value = getJsonValue(mapperModel, context);
 		if (value != null) {
-			context.setUserAttribute(attribute, value);
+			context.setUserAttribute(attribute, value.toString());
```

Comment:
I don't mind either way. If you like to fix it, please do, it should be a straightforward change.